### DISCUSSION TITLE
Is it possible to disable web interface?

### DISCUSSION THREAD

**Discussion by Bammede** [AdguardTeam/AdGuardHome#4172] [Q&A] [answered]
Is it possible to disable web interface, but keep Adguard Home work normally?

**Comment by ainar-g** [comment, marked as answer]
Currently, no.  But this is planned for v0.108.

**Reply by qosmio** [reply]
Follow up question, will it be possible to decouple the web interface?  I use AGH in my custom build of Openwrt, and it’s a significant part of the final binary. I can get it down to 10MB UPX compressed from 44MB, but it’s over 60% of the final size.

Would be great if I could separate them into separate packages so I can use an external usb to serve the UI but let the “core” functionality be imbedded.

**Reply by ainar-g** [reply]
@qosmio, sorry about the late response.  Something like that could be done through plugins or dynamic linking, but that would require a significant change in the architecture of AGH.  There is already an issue about reducing the size of the binary, see #4025.

**Comment by duhow** [comment]
Just hit by this topic, hoping this is addressed soon... 🙏🏻

**Comment by ghost** [comment]
Bump
